core/src/test/java/org/apache/calcite/test/TypeCoercionTest.java

```diff
@@ -253,17 +254,17 @@ private static ImmutableList<RelDataType> combine(
 
     // intersect
     sql("select t1_int, t1_decimal, t1_smallint, t1_double from t1 "
-        + "intersect select t2_varchar20, t2_decimal, t2_float, t2_bigint from t2 ")
```

Comment:
Why did we change all `float` type to `real` type?

Reply:
Because float and double are actually the same type, so the tests have better coverage now.
https://issues.apache.org/jira/browse/CALCITE-6074